roles/update/tasks/main.yml

```diff
@@ -44,11 +44,76 @@
     - cifmw_ci_gen_kustomize_values_installplan_approval is defined
     - cifmw_ci_gen_kustomize_values_installplan_approval | lower == 'manual'
 
+- name: Handle OpenStack operator initialization
+  when:
+    - cifmw_ci_gen_kustomize_values_deployment_version is defined
+    - cifmw_ci_gen_kustomize_values_deployment_version in ['v1.0.3', 'v1.0.6']
```

Comment:
Why do you use these specific versions? Please change hardcoded values to configurable arguments.

Reply:
This is because those are the only versions that will ever need that.  Starting from v1.0.7 we don't need that block.  So we might as well save us one variable and just hard code it.

Reply:
Ack. That makes sense.